test/jobs/cache/browser_channels/responses_for_prefix_test.rb

```diff
@@ -20,10 +20,28 @@ def self.test_order
     assert service.temp_file.present?
     result = Brotli.inflate(File.open(service.temp_file.path, 'rb').readlines.join("").slice(4..-1))
     result = PublishersPb::ChannelResponseList.decode(result)
+    assert result.channel_responses[0].wallets[0].uphold_wallet.address
```

Comment:
```suggestion
```

The next line already does a null check, so this isn't needed

Reply:
It's not a null check, it's an equality check. They could both be null.